### PR TITLE
AOO-24: stabilized latest branch

### PR DESCRIPTION
**Pull request by Elelan**
- removed unavailable libs and refactored their usage
1. com.github.stfalcon:frescoimageviewer
2. com.amulyakhare:com.amulyakhare.textdrawable

### CODE REVIEW

File: app/src/main/java/net/opendasharchive/openarchive/db/Space.kt [hunk 1]
```diff
@@ -164,7 +177,11 @@ data class Space(
 
             Type.GDRIVE -> ContextCompat.getDrawable(context, R.drawable.logo_gdrive_outline) // ?.tint(color)
 
-            else -> TextDrawable.builder().buildRound(initial, color)
+            else -> {
+                // Replace the original TextDrawable code with the custom drawable function
+                val initial = "A" // Replace with actual initial if available
```
Comment (prathieshna) [inline app/src/main/java/net/opendasharchive/openarchive/db/Space.kt:182]: Shall we remove this line

**Review by navodaSathsarani — APPROVED** [review]
LGTM

**Review by prathieshna — CHANGES_REQUESTED** [review]
Please check on the changes.

**Review by prathieshna — APPROVED** [review]
LGTM

**Pull request merged**